azure-pipelines.yml

```diff
@@ -213,18 +208,26 @@ stages:
             sonar.analysis.sha1=$(Build.SourceVersion)
             sonar.analysis.repository=$(Build.Repository.ID)
 
-      - task: VSBuild@1
-        displayName: 'Build SonarAnalyzer solution'
-        inputs:
-          solution: '$(solution)'
-          platform: '$(buildPlatform)'
-          configuration: '$(buildConfiguration)'
-          msbuildArgs: '/p:DeployExtension=false /p:Sha1=$(Build.SourceVersion) /p:BuildNumber=$(Build.BuildId)'
-
       - powershell: |
-          cd sonaranalyzer-dotnet\tests
-          $(NUGET_PACKAGES)\opencover\4.7.922\tools\OpenCover.Console.exe -target:"C:/Program Files/dotnet/dotnet.exe" -targetargs:"test SonarAnalyzer.UnitTest\bin\$(BuildConfiguration)\net48\SonarAnalyzer.UnitTest.dll" -returntargetcode -filter:"+[SonarAnalyzer.CFG]* +[SonarAnalyzer]* +[SonarAnalyzer.CSharp]* +[SonarAnalyzer.Utilities]* +[SonarAnalyzer.VisualBasic]*" -output:coverage.xml -oldStyle -register:Path32
-          cd ..
+          cd sonaranalyzer-dotnet
+
+          $testProjFileName = "tests\SonarAnalyzer.UnitTest\SonarAnalyzer.UnitTest.csproj"
+
+          Write-Host "Add .Net 5 target for UnitTest project"
+          ((Get-Content -Path $testProjFileName -raw) -Replace "net48;netcoreapp3.1","net48;netcoreapp3.1;net5") | Set-Content -Path $testProjFileName
+
+          Write-Host "Build UnitTest project"
+          dotnet build $testProjFileName -c $(BuildConfiguration)
+
+          Write-Host "Run tests on .Net 4.8 and compute coverage"
```

Comment:
on second thought - this is confusing - it says running tests on .NET 4.8 but it's using core dotnet 3.1

```
Run tests on .Net 4.8 and compute coverage
Executing: C:\Program Files\dotnet\dotnet.exe

Welcome to .NET Core 3.1!
---------------------
SDK Version: 3.1.301
```

could you improve this message in the logs?

Reply:
it seems it's still there - [see lines 2933-2939](https://sonarsource.visualstudio.com/DotNetTeam%20Project/_build/results?buildId=19772&view=logs&jobId=645a1db5-d210-5b98-34db-87d2a8fd8bb0&j=23c99647-d7e1-5b83-b5fb-f7e97f8a9ce7&t=0075a4ce-4d44-5a4c-36ef-97ab11b0917b)

Reply:
I've also noticed that. Not sure why since I've used `--nologo` and for the others it works. Not sure also how we can get rid of it.

Reply:
well meanwhile you can modify the message to say that it's testing the net48 binary (running on .net core 3.1)